CharacterClient/main.py

```
@@ -0,0 +1,66 @@
+        help="Access token for server authentication (default: env CLIENT_TOKEN)"
+    )
+    parser.add_argument(
+        "--client_host",
+        type=str,
+        default=os.getenv("CLIENT_HOST", "0.0.0.0"),
+        help="Host for the client's own API server (default: env CLIENT_HOST or 0.0.0.0)"
+    )
+    parser.add_argument(
+        "--client_port",
+        type=int,
+        default=int(os.getenv("CLIENT_PORT", 8001)), # Default to 8001 to avoid conflict with server
```

Pylint (3.3.7)

[warning] os.getenv default type is builtins.int. Expected str or None.

(W1508)